agent/cmd/cuda-vmm-interpose/interpose.c

```
@@ -0,0 +1,1779 @@
+}
+
+static bool
+is_logical_handle(CUmemGenericAllocationHandle handle)
+{
+  return ((uint64_t)handle & LOGICAL_HANDLE_TAG_MASK) == LOGICAL_HANDLE_TAG;
+}
+
+static struct handle*
+resolve_managed_handle(CUmemGenericAllocationHandle logical)
+{
+  struct handle* handle;
```

Clang (14.0.6)

[warning] variable 'handle' is not initialized

(cppcoreguidelines-init-variables)